src/index.jsx

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { render } from 'react-dom';
 import './style.css';
+import AnimalList from './components/AnimalList';
+import AnimalDetail from './components/AnimalDetail';
 
 const App = () => {
+
+  const[appData, setAppData] = useState(["wtf0", 'wtf1', "wtf2", "wtf"]);
+  
+  useEffect(
+    ()=>{
+      fetch('https://lrolecek.github.io/zviratka-api/zvirata.json')
+      .then( res => res.json())
+      .then((data)=>setAppData(data.zvirata))
+    },
+    []
+  )
+
+  const showDetail = (animal) =>{
+    setSelectedAnimal(animal)
+  }
+  const[selectedAnimal, setSelectedAnimal] = useState(null)
```

Comment:
typicky chceme mít všechny useState u sebe, tj dal bych to pod řádek 9, pod
`const[appData, setAppData] = useState(["wtf0", 'wtf1', "wtf2", "wtf"]);`